README.md

```diff
@@ -163,6 +163,16 @@ This could happen for a variety of reasons. The most likely of which is that one
 
 Per [#36](https://github.com/jeremyckahn/chitchatter/issues/36), check your `about:config` settings and ensure that `media.peerconnection.enabled` is **enabled**.
 
+### Contributors
```

Comment:
![image](https://user-images.githubusercontent.com/85288261/206241932-6d4cb509-ac78-4889-8ff5-f08166fb5993.png)

contrib.rocks

Reply:
in readme preview 

![image](https://user-images.githubusercontent.com/85288261/206908599-4ab035df-2d33-4827-af61-80436e088dba.png)

Reply:
[visit](https://github.com/MohammadTaseenKhan/chitchatter/tree/patch-2#contributors)